libyul/AsmJsonImporter.cpp

```diff
@@ -157,7 +157,12 @@ Literal AsmJsonImporter::createLiteral(Json::Value const& _node)
 	auto lit = createAsmNode<Literal>(_node);
 	string kind = member(_node, "kind").asString();
 
-	lit.value = YulString{member(_node, "value").asString()};
+	solAssert(member(_node, "hexValue").isString() || member(_node, "value").isString(), "");
+	if (_node.isMember("hexValue"))
+		lit.value = YulString{util::asString(util::fromHex(member(_node, "hexValue").asString()))};
```

Comment:
Wouldn't it make sense to also use `util::validateUTF8` here?

Reply:
No, why? Literals can be invalid utf8